agent/engine/docker_container_engine.go

```diff
@@ -117,6 +118,36 @@ func NewDockerGoClient() (*DockerGoClient, error) {
 	}, nil
 }
 
+func (dg *DockerGoClient) ImportImage(name string, tag string, reader io.Reader) DockerContainerMetadata {
+	timeout := ttime.After(pullImageTimeout)
+	response := make(chan DockerContainerMetadata, 1)
+
+	// Don't import the same image twice.
+	_, err := dg.dockerClient.InspectImage(name + ":" + tag)
```

Comment:
Fixed with this [commit](https://github.com/swipely/amazon-ecs-agent/commit/9241c665ab58521b9eb29a4f41efe8e1d5a70396).